docker/dev/alpine/Dockerfile-opflex-build-base

```diff
@@ -8,18 +8,17 @@ RUN apk upgrade --no-cache && apk add --no-cache build-base \
     libnetfilter_conntrack-dev rapidjson-dev python3-dev bzip2-dev \
     curl libcurl curl-dev zlib-dev
 ARG make_args=-j4
-RUN git clone https://github.com/noironetworks/3rdparty-debian.git
-RUN git clone https://github.com/jupp0r/prometheus-cpp.git \
+RUN git clone https://github.com/noironetworks/3rdparty-debian.git \
+  && git clone https://github.com/jupp0r/prometheus-cpp.git \
```

Comment:
Can we fix the branch to v0.12.1 here as well?

Reply:
Good catch. I still have an issue with cmake in the container so I'll push another revision with both of those changes